.github/workflows/build-and-deploy.yml

```diff
@@ -427,7 +427,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        environment: [qa,staging,production,sandbox]
+        environment: [qa,staging,production,sandbox,rollover]
```

Comment:
I wonder if we should not be automatically deploying to rollover each time, but add it to the deploy workflow to do this manually to avoid slowing down builds 🤔  what are your thoughts